dist/Tie-File/lib/Tie/File.pm

```diff
@@ -757,7 +762,7 @@ sub _oadjust {
     # to the first new one of this batch
     for my $i ($prev_end+2 .. $pos - 1) {
       $self->{offsets}[$i] += $delta;
-      $newkey{$i} = $i + $delta_recs;
+      $newkeys{$i} = $i + $delta_recs;
```

Comment:
This code was previously using the wrong variable name. Until I added strict, it was not obvious there was a problem. 

I am disturbed this didn't change the results of the tests in any way.

Reply:
Looks like %newkeys wasn't even being used. I have removed the code in its own commit.